cmd/osmosisd/cmd/root.go

```diff
@@ -1328,10 +1328,5 @@ func autoCliOpts(initClientCtx client.Context, tempApp *osmosis.OsmosisApp) auto
 		AddressCodec:          authcodec.NewBech32Codec(sdk.GetConfig().GetBech32AccountAddrPrefix()),
 		ValidatorAddressCodec: authcodec.NewBech32Codec(sdk.GetConfig().GetBech32ValidatorAddrPrefix()),
 		ConsensusAddressCodec: authcodec.NewBech32Codec(sdk.GetConfig().GetBech32ConsensusAddrPrefix()),
-		ClientCtx:             initClientCtx,
-		TxConfigOpts: tx.ConfigOptions{
-			EnabledSignModes:           tx.DefaultSignModes,
-			TextualCoinMetadataQueryFn: authtxconfig.NewGRPCCoinMetadataQueryFn(initClientCtx),
-		},
-	}
```

Comment:
This is safe to delete, txconfigOpts was deprecated with cosmos sdk client v2.0.0-beta.3 bump. Tested with CLI to confirm